app/services/emergency_manager.py

```
@@ -0,0 +1,620 @@
+                        "symbol": position["symbol"],
+                        "original_quantity": original_qty,
+                        "reduced_quantity": reduced_qty,
+                        "reduction_pct": reduction_pct,
+                        "execution_result": reduction_result
+                    })
+            
+            return {
+                "success": True,
+                "actions_taken": actions_taken,
+                "message": f"Reduced all positions by {reduction_pct}%"
+            }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)